src/loaders/mod_load.c

```diff
@@ -80,7 +80,6 @@ const struct mod_magic mod_magic[] = {
 	{"FA06", 1, TRACKER_DIGITALTRACKER, 6},	/* Atari Falcon */
 	{"FA08", 1, TRACKER_DIGITALTRACKER, 8},	/* Atari Falcon */
 	{"NSMS", 1, TRACKER_UNKNOWN, 4},	/* in Kingdom.mod */
-	{"", 0}
```

Comment:
you could just complete the initializer, but OK

Reply:
I didn't really see the point in using sentinel termination for this list when it is static, fixed length, and accessible to every place that uses it. I had to change it anyway, so I went with the option I thought was better. Don't know that it saves any time/space, but it seems conceptually cleaner to me.

Also, I accidentally leave off array sentinels sometimes. :(